### ISSUE TITLE
Tests failed: Matrices are different at the precision

### ISSUE THREAD

**Issue by sagitter** [openmeeg/openmeeg#266]
Hello everyone.

I'm compiling `OpenMEEG` on Fedora ([build log](https://kojipkgs.fedoraproject.org//work/tasks/2198/25732198/build.log)), some tests are failing because of this error:

`504: ERROR : Matrices are different at the precision : eps = 0.1`

The following tests FAILED:

```
	465 - cmp-EEGEST-dip-HeadMN1-dip1-rdm (Failed)
	466 - cmp-EEGadjointEST-dip-HeadMN1-dip1-rdm (Failed)
	467 - cmp-EEGadjoint2EST-dip-HeadMN1-dip1-rdm (Failed)
	471 - cmp-EEGEST-dip-HeadMN2-dip1-rdm (Failed)
	472 - cmp-EEGadjointEST-dip-HeadMN2-dip1-rdm (Failed)
	473 - cmp-EEGadjoint2EST-dip-HeadMN2-dip1-rdm (Failed)
	474 - cmp-EEGEST-dip-HeadMN1-dip2-mag (Failed)
	475 - cmp-EEGadjointEST-dip-HeadMN1-dip2-mag (Failed)
	476 - cmp-EEGadjoint2EST-dip-HeadMN1-dip2-mag (Failed)
	477 - cmp-EEGEST-dip-HeadMN1-dip2-rdm (Failed)
	478 - cmp-EEGadjointEST-dip-HeadMN1-dip2-rdm (Failed)
	479 - cmp-EEGadjoint2EST-dip-HeadMN1-dip2-rdm (Failed)
	480 - cmp-EEGEST-dip-HeadMN2-dip2-mag (Failed)
	481 - cmp-EEGadjointEST-dip-HeadMN2-dip2-mag (Failed)
	482 - cmp-EEGadjoint2EST-dip-HeadMN2-dip2-mag (Failed)
	483 - cmp-EEGEST-dip-HeadMN2-dip2-rdm (Failed)
	484 - cmp-EEGadjointEST-dip-HeadMN2-dip2-rdm (Failed)
	485 - cmp-EEGadjoint2EST-dip-HeadMN2-dip2-rdm (Failed)
	486 - cmp-EEGEST-dip-HeadMN1-dip3-mag (Failed)
	487 - cmp-EEGadjointEST-dip-HeadMN1-dip3-mag (Failed)
	488 - cmp-EEGadjoint2EST-dip-HeadMN1-dip3-mag (Failed)
	489 - cmp-EEGEST-dip-HeadMN1-dip3-rdm (Failed)
	490 - cmp-EEGadjointEST-dip-HeadMN1-dip3-rdm (Failed)
	491 - cmp-EEGadjoint2EST-dip-HeadMN1-dip3-rdm (Failed)
	492 - cmp-EEGEST-dip-HeadMN2-dip3-mag (Failed)
	493 - cmp-EEGadjointEST-dip-HeadMN2-dip3-mag (Failed)
	494 - cmp-EEGadjoint2EST-dip-HeadMN2-dip3-mag (Failed)
	495 - cmp-EEGEST-dip-HeadMN2-dip3-rdm (Failed)
	496 - cmp-EEGadjointEST-dip-HeadMN2-dip3-rdm (Failed)
	497 - cmp-EEGadjoint2EST-dip-HeadMN2-dip3-rdm (Failed)
	498 - cmp-EEGEST-dip-HeadMN1-dip4-mag (Failed)
	499 - cmp-EEGadjointEST-dip-HeadMN1-dip4-mag (Failed)
	500 - cmp-EEGadjoint2EST-dip-HeadMN1-dip4-mag (Failed)
	501 - cmp-EEGEST-dip-HeadMN1-dip4-rdm (Failed)
	502 - cmp-EEGadjointEST-dip-HeadMN1-dip4-rdm (Failed)
	503 - cmp-EEGadjoint2EST-dip-HeadMN1-dip4-rdm (Failed)
	504 - cmp-EEGEST-dip-HeadMN2-dip4-mag (Failed)
	505 - cmp-EEGadjointEST-dip-HeadMN2-dip4-mag (Failed)
	506 - cmp-EEGadjoint2EST-dip-HeadMN2-dip4-mag (Failed)
	507 - cmp-EEGEST-dip-HeadMN2-dip4-rdm (Failed)
	508 - cmp-EEGadjointEST-dip-HeadMN2-dip4-rdm (Failed)
	509 - cmp-EEGadjoint2EST-dip-HeadMN2-dip4-rdm (Failed)
	510 - cmp-EEGEST-dip-HeadMN1-dip5-mag (Failed)
	511 - cmp-EEGadjointEST-dip-HeadMN1-dip5-mag (Failed)
	512 - cmp-EEGadjoint2EST-dip-HeadMN1-dip5-mag (Failed)
	513 - cmp-EEGEST-dip-HeadMN1-dip5-rdm (Failed)
	514 - cmp-EEGadjointEST-dip-HeadMN1-dip5-rdm (Failed)
	515 - cmp-EEGadjoint2EST-dip-HeadMN1-dip5-rdm (Failed)
	516 - cmp-EEGEST-dip-HeadMN2-dip5-mag (Failed)
	517 - cmp-EEGadjointEST-dip-HeadMN2-dip5-mag (Failed)
	518 - cmp-EEGadjoint2EST-dip-HeadMN2-dip5-mag (Failed)
	519 - cmp-EEGEST-dip-HeadMN2-dip5-rdm (Failed)
	520 - cmp-EEGadjointEST-dip-HeadMN2-dip5-rdm (Failed)
	521 - cmp-EEGadjoint2EST-dip-HeadMN2-dip5-rdm (Failed)
	525 - cmp-EEGEST-dip-HeadMN1-dip6-rdm (Failed)
	526 - cmp-EEGadjointEST-dip-HeadMN1-dip6-rdm (Failed)
	527 - cmp-EEGadjoint2EST-dip-HeadMN1-dip6-rdm (Failed)
	531 - cmp-EEGEST-dip-HeadMN2-dip6-rdm (Failed)
	532 - cmp-EEGadjointEST-dip-HeadMN2-dip6-rdm (Failed)
	533 - cmp-EEGadjoint2EST-dip-HeadMN2-dip6-rdm (Failed)
```

**Comment by agramfort** [comment]
hi @sagitter 

can you try this PR https://github.com/openmeeg/openmeeg/pull/265

we have a green travis using ubuntu so there is a good chance it works on your fedora too.

let us know. thx

**Comment by sagitter** [comment]
Fixed on ee565c437650f11786f336a1f620474d31e64018.
